src/components/ImageSlab/ImageSlab.tsx

```diff
@@ -0,0 +1,38 @@
+import CustomImage from "../CustomImage/CustomImage";
+
+interface ImageData {
+  imageSrc: string;
+  imageAltText: string;
+  className?: string;
+}
+
+interface Props {
+  imageData: ImageData;
+  slabSize: string;
```

Comment:
this is a good place to use an enum:

```tsx
export enum SlabSize {
  MOBILE = 'mobile',
  TABLE = 'tablet,
  DESKTOP = 'desktop',
  THUMBNAIL = 'thumbnail',
}

// and then for the prop type:

  slabSize: SlabSize;
```

the enum is exported so that it can be used by whatever components render an `ImageSlab`